strings/base_coroutine_foundation.h

```diff
@@ -801,4 +801,43 @@ WINRT_EXPORT namespace winrt
         impl::check_status_canceled(shared->status);
         co_return shared->result.GetResults();
     }
+
+    template <typename Delegate>
+    struct async_event : impl::event_base<Delegate>
```

Comment:
Trying to think of a clearer name. Maybe `background_invoked_event`? The point is that the event handlers are invoked on a (new) background thread.